### ISSUE TITLE
Outdated installation documentation in the Platform user manual

### ISSUE THREAD

**Issue by tuxpiper** [ushahidi/platform#3687]
Thanks @tuxpiper, in fact I've moved to 4.0.0 now, and the packaged phinx version is 0.8.1.
```
# ./bin/phinx -V
Phinx by Rob Morgan - https://phinx.org. 0.8.1
```

I did a clean install with Ushahidi v4 with a new DB, and I came across at least 2 issues:

- `Fatal error: Class 'Memcached' not found` when attempting setup DB migration
Fix: need to specify CACHE_DRIVER=file in .env
This is *not* specified in the docs, either in the release README or the website manual at https://docs.ushahidi.com/ushahidi-platform-user-manual/1.-installing-ushahidi/1.1-installing-the-latest-release

- `each() function is deprecated`
I am using PHP 7.2 which is standard on Ubuntu 18.04 and is apparently supported according to the docs.
However I have to do the hackfix at https://github.com/ushahidi/platform-release/issues/10#issuecomment-415334098 in order to even get ./bin/ushahidi to run from the cmdline.

Re docs and versioning:
I appreciate 4.0.0 is the latest release, but this is very inconsistently tracked in the docs.
The main section of the Ushahidi User Manual starts with a section titled: "Overview of Ushahidi Platform v3.0"(https://docs.ushahidi.com/ushahidi-platform-user-manual/overview-of-ushahidi-platform-v3.0). There are many such examples throughout the docs and also intermingled references to v2. 
Furthermore there are several conflicts between the README and the online installation manual.
For example:
Website: `./bin/phinx migrate -c application/phinx.php`
README: `./bin/phinx migrate -c phinx.php`
Yet more confusingly, some critical information are only in the [platform-release Release Notes](https://github.com/ushahidi/platform-release/releases/tag/v4.0.0) for 4.0.0, and **neither** in the release README **nor** in the online manual.
Eg Updated Database tokens ie `DB_CONNECTION=mysql`
The latest platform release is also v4.0 but the API version is apparently still v3 as reflected by the routes.

Currently, despite the above, I can't get my platform-release to load - I keep getting the error:

> Sorry, something went wrong. Try reloading the page.

in the browser and the error `TypeError: "e.data.errors is undefined"` in the browser console.
Kinda frustrating as I've been at this a while now. What am I doing wrong?

_Originally posted by @danryu in https://github.com/ushahidi/platform/issues/2705#issuecomment-527412067_

**Comment by tuxpiper** [comment]
Writing as I'm chipping away stuff from @danryu 's ⭐️  comments:

> each() function is deprecated
I am using PHP 7.2 which is standard on Ubuntu 18.04 and is apparently supported according to the docs.
However I have to do the hackfix at ushahidi/platform-release#10 (comment) in order to even get ./bin/ushahidi to run from the cmdline.

This will be solved in the upcoming release that we are polishing now. Aura-DI will be 3.4 .

The docs should have never mentioned PHP 7.2 support, we must have had a confusion or mistake with that at some point.

For the next upcoming release, things are looking good for PHP 7.2 and may I even say PHP 7.3 support in the release. But we will confirm in the changelog and documentation once everything is tested and done.

We are also making PHP major release support test a routine in the automated tests ( https://github.com/ushahidi/platform/pull/3678 ) , so we don't repeat this confusion again.

> Fatal error: Class 'Memcached' not found when attempting setup DB migration
Fix: need to specify CACHE_DRIVER=file in .env
This is not specified in the docs, either in the release README or the website manual at https://docs.ushahidi.com/ushahidi-platform-user-manual/1.-installing-ushahidi/1.1-installing-the-latest-release
 > Furthermore there are several conflicts between the README and the online installation manual.
For example:
Website: ./bin/phinx migrate -c application/phinx.php
README: ./bin/phinx migrate -c phinx.php
Yet more confusingly, some critical information are only in the platform-release Release Notes for 4.0.0, and neither in the release README nor in the online manual.
Eg Updated Database tokens ie DB_CONNECTION=mysql
The latest platform release is also v4.0 but the API version is apparently still v3 as reflected by the routes.

Tearing down those outdated docs and updating the README for github.com/usahidi/platform-release to point at the latest and greatest docs . Sorry for the confusion :(

The good docs to install the platform release package: https://docs.ushahidi.com/platform-developer-documentation/getting-started/setup_alternatives/platform_release_install

> Currently, despite the above, I can't get my platform-release to load - I keep getting the error:
    Sorry, something went wrong. Try reloading the page.
in the browser and the error TypeError: "e.data.errors is undefined" in the browser console.
Kinda frustrating as I've been at this a while now. What am I doing wrong?

There were a couple things missing in the docs and that we solved via chat. I made sure they are reflected in the good docs now.
